### DISCUSSION TITLE
Footnote References

### DISCUSSION THREAD

**Discussion by robertnwinter** [Zettlr/Zettlr#2983] [Q&A]
Hello All,

I upgraded to version 2.1 yesterday and noticed an odd behaviour with reference styles that use footnotes instead of in text. 

In v. 2.0.3, when choosing a footnote reference style (e.g. oxford-art-journal.csl), footnotes were created on export. In 2.1, while the footnotes are being created, the author name remains in text just before the footnote. 

I think this is a bug, unless it is something unique to my installation. I am running Zettlr on Fedora.

Also, is there a link to the old 2.0.3 version as I would like to revert to an older install until I have found a resolution. 

Kind Regards,

**Comment by nathanlesage** [comment]
> Also, is there a link to the old 2.0.3 version as I would like to revert to an older install until I have found a resolution.

You can find all previous releases here, on the Releases tab. Converting to issue …

**Reply by nathanlesage** [reply]
Just tried this out and I could not reproduce it with the specific CSL style you suggested (see screenshot) and on 2.1 -- but on the current develop branch. So it might've been a minor bug that got fixed by itself. Feel free to try out today's nightlies and see if the bug still occurs. Otherwise: There have been some changes to Pandoc's citation handling between 2.0.3 and 2.1.0, so maybe something that wasn't supposed to work before is not working anymore …?

![image](https://user-images.githubusercontent.com/17251683/147961842-476004b5-3359-462b-9e03-7db20bc26217.png)

**Reply by robertnwinter** [reply]
Many thanks for the prompt response. 

I have done a little more testing and found the root cause, it is how auto-complete is working with the references. So long as references are fully enclosed within brackets (e.g. [@burkeEarlyLifeCorrespondence2014, 23] ) then the application works as expected with the footnote being placed after the punctuation as a footnote. 

However, if I choose auto-complete from an @ with no bracket (e.g. @burkeEarlyLifeCorrespondence2014 [23] ), then this is when the author name is left in the sentence:

![Untitled](https://user-images.githubusercontent.com/95027082/148138956-2f5123f3-9869-4b6b-8ecb-052a658ec2da.jpg)

I am getting this behaviour with 2.0.3 and 2.1. So I think I have only noticed this problem because I have traditionally always used the brackets. I only changed this method recently as on some APA documents I wanted a sentence such as 'Burke (1790) notes that...'

**Reply by nathanlesage** [reply]
Alright, that's not a bug, that's a feature :D If you choose the first option, then it should cite as you would like to have it. Basically, the settings description above works for footnote-styles in such a way that everything that's in parenthesis will form the body of a footnote. This is so that citeproc can accomodate different ways of citing.

**Reply by robertnwinter** [reply]
Haha! Thought it may be the case of a feature **not** a bug. Having played with the functionality, I can definitely see why it is this way, and why its a great feature. 

But I was less jubilant when I was steadily replacing the first ~200 references in my thesis after discovering the feature... ;-)
